util/libcrypto.num

```diff
@@ -4218,3 +4218,4 @@ BIO_meth_get_write_ex                   4168	1_1_1	EXIST::FUNCTION:
 BIO_meth_set_write_ex                   4169	1_1_1	EXIST::FUNCTION:
 DSO_pathbyaddr                          4170	1_1_0c	EXIST::FUNCTION:
 DSO_dsobyaddr                           4171	1_1_0c	EXIST::FUNCTION:
+OCSP_resp_get1_id                       4172	1_1_1	EXIST::FUNCTION:OCSP
```

Comment:
If this is going to be backported to 1.1.0 then the symbol version needs to be 1_1_0d (i.e. the next 1.1.0 release version). Also this needs rebasing because there is a conflict in this file.